### PR TITLE
Added Go gRPC snippets

### PR DESCRIPTION
**Pull request by oskardudycz**
Adds snippets for https://github.com/EventStore/EventStore-Client-Go/pull/65.

### CODE REVIEW

**Review by mat-mcloughlin — APPROVED** [review]

**Comment by mat-mcloughlin** [comment]
Looks good but I can't speak to the correctness of the go samples

File: docs/clients/grpc/README.md [hunk 1]
```diff
@@ -42,7 +42,13 @@ implementation 'com.eventstore:db-client-java:0.5'
 libraryDependencies += "com.eventstore" % "db-client-java" % "0.6"
 ```
 :::
-::: code-group-item JavaScript
+
+::: code-group-item Go
```
Comment (ylorph) [inline docs/clients/grpc/README.md]: this section should go above c# & Java (up a fex line )

File: docs/clients/grpc/projections.md [hunk 2]
```diff
@@ -76,6 +79,11 @@ Projection management operations are exposed through a dedicated client.
 Sample available soon
 ```
 
+</xode-block>
+<xode-block title="Go" code="connectionString">
```
Comment (ylorph) [inline docs/clients/grpc/projections.md]: needs a line feed to render correctly
this while block needs to be befor the java one

File: docs/clients/grpc/projections.md [hunk 3]
```diff
@@ -76,6 +79,11 @@ Projection management operations are exposed through a dedicated client.
...
+```
+// This is currently not available in the Go client
+```
 </xode-block>
```
Comment (ylorph) [inline docs/clients/grpc/projections.md]: needs a line feed just before this line to render correctly

File: docs/clients/grpc/projections.md [hunk 4]
```diff
@@ -236,9 +258,12 @@ The .net clients up to version 21.2 had an incorrect behavior: they _will not_ s
 :::
 
 :::: code-group
-::: code-group-item c#
+::: code-group-item C#
 @[code{Disable}](../dotnet/21.6/samples/projection-management/Program.cs)
 :::
+::: code-group-item Go
+// This is currently not available in the Go client
```
Comment (ylorph) [inline docs/clients/grpc/projections.md:279]:
```suggestion
```
// This is currently not available in the Go client
```
```

File: docs/clients/grpc/projections.md [hunk 5]
```diff
@@ -260,9 +285,12 @@ The .net clients up to version 21.2 had an incorrect behavior: they _will not_ s
 You can only disable an existing projection. When you try to disable a non-existing projection, you'll get an error:
 
 :::: code-group
-::: code-group-item c#
+::: code-group-item C#
 @[code{DisableNotFound}](../dotnet/21.6/samples/projection-management/Program.cs)
 :::
+::: code-group-item Go
+// This is currently not available in the Go client
```
Comment (ylorph) [inline docs/clients/grpc/projections.md:308]:
```suggestion
```
// This is currently not available in the Go client
```
```

File: docs/clients/grpc/projections.md [hunk 6]
```diff
@@ -286,11 +314,14 @@ You can only disable an existing projection. When you try to disable a non-exist
...
 
 :::: code-group
-::: code-group-item c#
+::: code-group-item C#
 ```csharp
 // This is currently not available in the .net client
 ```
 :::
+::: code-group-item Go
+// This is currently not available in the Go client
```
Comment (ylorph) [inline docs/clients/grpc/projections.md:341]:
```suggestion
```
// This is currently not available in the Go client
```
```

File: docs/clients/grpc/projections.md [hunk 7]
```diff
@@ -312,11 +343,14 @@ Deletes a projection
...
```
Comment (ylorph) [inline docs/clients/grpc/projections.md:372]:
```suggestion
```
// This is currently not available in the Go client
```
```

File: docs/clients/grpc/projections.md [hunk 8]
```diff
@@ -344,9 +378,12 @@ The .net clients up to version 21.2 had an incorrect behavior: they _will_ save
 :::
 
 :::: code-group
-::: code-group-item c#
+::: code-group-item C#
 @[code{Abort}](../dotnet/21.6/samples/projection-management/Program.cs)
 :::
+::: code-group-item Go
+// This is currently not available in the Go client
```
Comment (ylorph) [inline docs/clients/grpc/projections.md:407]:
```suggestion
```
// This is currently not available in the Go client
```
```

File: docs/clients/grpc/projections.md [hunk 9]
```diff
@@ -368,9 +405,12 @@ The .net clients up to version 21.2 had an incorrect behavior: they _will_ save
 You can only abort an existing projection. When you try to abort a non-existing projection, you'll get an error:
 
 :::: code-group
-::: code-group-item c#
+::: code-group-item C#
 @[code{Abort_NotFound}](../dotnet/21.6/samples/projection-management/Program.cs)
 :::
+::: code-group-item Go
+// This is currently not available in the Go client
```
Comment (ylorph) [inline docs/clients/grpc/projections.md:436]:
```suggestion
```
// This is currently not available in the Go client
```
```

File: docs/clients/grpc/projections.md [hunk 10]
```diff
@@ -394,9 +434,12 @@ You can only abort an existing projection. When you try to abort a non-existing
 Resets a projection. This will re-emit events. Streams that are written to from the projection will also be soft deleted.
 
 :::: code-group
-::: code-group-item c#
+::: code-group-item C#
 @[code{Reset}](../dotnet/21.6/samples/projection-management/Program.cs)
 :::
+::: code-group-item Go
+// This is currently not available in the Go client
```
Comment (ylorph) [inline docs/clients/grpc/projections.md:467]:
```suggestion
```
// This is currently not available in the Go client
```
```

File: docs/clients/grpc/projections.md [hunk 11]
```diff
@@ -418,9 +461,12 @@ Resets a projection. This will re-emit events. Streams that are written to from
 Resetting a projection that does not exists will result in an error.
 
 :::: code-group
-::: code-group-item c#
+::: code-group-item C#
 @[code{Reset_NotFound}](../dotnet/21.6/samples/projection-management/Program.cs)
 :::
+::: code-group-item Go
+// This is currently not available in the Go client
```
Comment (ylorph) [inline docs/clients/grpc/projections.md:496]:
```suggestion
```
// This is currently not available in the Go client
```
```

File: docs/clients/grpc/projections.md [hunk 12]
```diff
@@ -445,9 +491,12 @@ Resetting a projection that does not exists will result in an error.
 Updates a projection. The name parameter is the name of the projection to be updated. The query parameter contains the new JavaScript.
 
 :::: code-group
-::: code-group-item c#
+::: code-group-item C#
 @[code{Update}](../dotnet/21.6/samples/projection-management/Program.cs)
 :::
+::: code-group-item Go
+// This is currently not available in the Go client
```
Comment (ylorph) [inline docs/clients/grpc/projections.md:528]:
```suggestion
```
// This is currently not available in the Go client
```
```

File: docs/clients/grpc/projections.md [hunk 13]
```diff
@@ -523,9 +578,12 @@ Returns a list of all continuous projections.
 See the [projection details](#projection-details) section for an explanation of the returned values
 
 :::: code-group
-::: code-group-item c#
+::: code-group-item C#
 @[code{ListContinuous}](../dotnet/21.6/samples/projection-management/Program.cs)
 :::
+::: code-group-item Go
+// This is currently not available in the Go client
```
Comment (ylorph) [inline docs/clients/grpc/projections.md:621]:
```suggestion
```
// This is currently not available in the Go client
```
```

**Review by ylorph — CHANGES_REQUESTED** [review]

**Comment by oskardudycz** [comment]
@ylorph, thank you for the review and finding those issues. I fixed all of them in https://github.com/EventStore/documentation/pull/450/commits/919328fb249ea14ca2a2da0561e11fe403186dad.

File: docs/clients/grpc/reading-events.md [hunk 14]
```diff
@@ -146,6 +158,9 @@ The simplest way to read the `$all` stream forwards is to supply a direction and
 ::: code-group-item C#
 @[code{read-from-all-stream}](../dotnet/21.2/samples/reading-events/Program.cs)
 :::
+::: code-group-item Go
+@[code{read-from-all-stream}](../go/1.0.0/samples/readingEvents.go)
+:::
```
Comment (ylorph) [inline docs/clients/grpc/reading-events.md:163]: do we have a sample for go line 180 ?

Comment (oskardudycz) [reply docs/clients/grpc/reading-events.md:163]: @ylorph, thanks for catching that. It is fixed in https://github.com/EventStore/documentation/pull/450/commits/15c40d572c6193473b281db6cd8c4d0b64a43c44.

**Review by ylorph — APPROVED** [review]

**Pull request merged**